ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
>Concerning the field appearing in the InstrumentContainer, I though this container will be >deprecated, but indeed if it is not I can use it

The main reason we say that InstrumentContainer will be deprecated is that it contains info that will not be in the data files, but rather in a central database with a separate API.  Anything that describes the instumental layout or configuration will be in this database, and not repeated in each data file as it is now.  For now, it's fine to keep using the insturment container until this new API is ready, though, and it may even help us to not miss something useful.